api/src/pcapi/routes/public/individual_offers/v1/bookings.py

```diff
@@ -220,9 +220,15 @@ def validate_booking_by_token(token: str) -> None:
 @api_key_required
 def cancel_booking_validation_by_token(token: str) -> None:
     """
-    Cancel a booking validation
+    Revert booking validation
 
-    Mark a booking as `unused`.
+    This operation reverses the status of a booking from `USED` back to `CONFIRMED`.
+    As a result, the pass Culture application will treat the booking as if the beneficiary has not retrieved it,
+    and the cultural partner will not receive a refund.
+
+    **⚠️ Warning:**
+    Before using this endpoint, **ensure that the cultural partner has manually confirmed their desire to revert the booking validation**.
+    Failure to do so may result in the cultural partner expecting a refund that will not be issued, due to the booking validation being reverted without their knowledge.
```

Comment:
```suggestion
    Failing to do so may result in the cultural partner expecting a refund that will not be issued, due to the booking validation being reverted without their knowledge.
```

Je ne suis pas sûr mais il me semble que `Failing` est plus approprié dans cette tournure.

Reply:
C'est chat GPT qui m'a proposé cette formule :D